node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -228,7 +230,7 @@ func (r OsDeviceConnectivityHelperScsiGeneric) FlushMultipathDevice(mpathDevice
 	logger.Debugf("Try to acquire lock for running the command multipath -f {%v} (to avoid concurrent multipath commands)", mpathDevice)
 	r.MutexMultipathF.Lock()
 	logger.Debugf("Acquired lock for multipath -f command")
-	_, err = r.Executer.ExecuteWithTimeout(TimeOutMultipathCmd, multipathCmd, []string{"-f", fullDevice})
+	_, err = r.Executer.ExecuteWithTimeout(TimeOutMultipathCmd, multipathCmd, []string{"-f", fullDevice}, false)
```

Comment:
this is how it looks:
```
... (device_connectivity_helper_scsigeneric.go:232) - Acquired lock for multipath -f command
... (executer.go:75) - Finished executing command
```
I think the log should explicitly state the full command that is about to be used, right before it is executed. ideally without maintaining a duplication of all its details (e.g. parameters) in a separate message.

i.e. I would remove the `false` here

Reply:
the full command is multipath -f

Reply:
this is how the full command looks like before this PR:
```
... (device_connectivity_helper_scsigeneric.go:230) - Acquired lock for multipath -f command
... (executer.go:46) - Executing command : {multipath} with args : {[-f /dev/dm-3]}. and timeout : {60000} mseconds
... (executer.go:67) - Output from command: 
... (executer.go:72) - Finished executing command
```
note the `dm-3` (/potentially other parameters in the future)

Reply:
I removed it from the Silent group and conditioned the logging of the output in case it's empty.l